src/foam/dao/jdbc/AbstractJDBCDAO.java

```diff
@@ -0,0 +1,145 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.dao.jdbc;
+
+import foam.dao.AbstractDAO;
+
```

Comment:
I prefer to just group all imports together without spaces between lines.

Reply:
ok